third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -721,8 +721,8 @@ pub enum StatusCode {
     DEPENDENCY_LIMIT_REACHED = 1124,
     // This error indicates that unstable bytecode generated by the compiler cannot be published to mainnet
     UNSTABLE_BYTECODE_REJECTED = 1125,
+    USER_DEFINED_NATIVE = 1126,
```

Comment:
Hmm... you might need to rebase your PR. 1126 is already taken by a new status code PROGRAM_TOO_COMPLEX

Reply:
Also wondering if we should change the native to something more descriptive, like, USER_NATIVE_NOT_ALLOWED?

Reply:
Done!